genic/power.c

```diff
@@ -92,7 +92,8 @@ void parse_transfer(int i, double k, char * line, struct table *out_tab, int * I
      * Everything is a ratio against tot*/
     out_tab->logD[0][i] = pow(transfers[1]/transfers[5],2);
     out_tab->logD[1][i] = pow(transfers[0]/transfers[5],2);
-    out_tab->logD[2][i] = pow(transfers[6]/transfers[5],2);
+    out_tab->logD[2][i] = pow(transfers[4]/transfers[5],2);
+    out_tab->logD[3][i] = pow(transfers[6]/transfers[5],2);
```

Comment:
Could you add a comment which column is which species?

Reply:
Done